src/screens/SendToken/SendTokenContacts.js

```diff
@@ -302,6 +299,11 @@ class SendTokenContacts extends React.Component<Props, State> {
         });
     }
 
+    // asset transfer between user accounts only in regular, but not in PPN send flow
+    if (!this.isPPNTransaction) {
+      contactsToRender = [...userAccounts, ...contactsToRender];
```

Comment:
That means even if I search for the contact username, I will still see my account on top in search results. Wouldn't be that confusing?

Reply:
Updated 👍